Cubical/Categories/DistLatticeSheaf/Extension.agda

```diff
@@ -0,0 +1,1011 @@
+                                               singCase
+            where
+            singCase : ∀ i → (h ⋆⟨ C ⟩ ⋁Pullback .pbPr₂) ⋆⟨ C ⟩ restCone β β∈L' .coneOut (sing i)
+                           ≡ g ⋆⟨ C ⟩ restCone β β∈L' .coneOut (sing i)
+            singCase i = ⋆Assoc C _ _ _ ∙ transp (λ 𝕚 → h ⋆⟨ C ⟩
+                 (++Lemmas.toConeOutPathPL ((pbPr₁ ⋁Pullback) ★ (restCone γ γ∈L'))
+                                           ((pbPr₂ ⋁Pullback) ★ (restCone β β∈L'))
+                                           (to++ConeSquare _ _ (pbCommutes ⋁Pullback)) i (~ 𝕚))
+                ≡ ++Lemmas.toConeOutPathPL (f ★ (restCone γ γ∈L'))
+                                           (g ★ (restCone β β∈L'))
+                                           (to++ConeSquare _ _ square) i (~ 𝕚)) i0 singCaseHelper
+              where
+              fromAssumption : h ⋆⟨ C ⟩ (coneOut (restCone (β ++Fin γ) (β++γ∈L' β∈L' γ∈L'))
+                                                 (sing (FSCfun _ _ (inl i))))
+                                    ≡ coneOut (toCone f g square) (sing (FSCfun _ _ (inl i)))
+              fromAssumption = hIsConeMor (sing (FSCfun _ _ (inl i)))
+
+              singCaseHelper :  h ⋆⟨ C ⟩ (coneOut ++LimCone' (sing (FSCfun _ _ (inl i))))
+                                    ≡ coneOut (toCone f g square) (sing (FSCfun _ _ (inl i)))
+              singCaseHelper = subst (λ x → h ⋆⟨ C ⟩ x ≡ coneOut (toCone f g square)
+                                                                 (sing (FSCfun _ _ (inl i))))
+                                     (sym (++LimCone≡ (FSCfun _ _ (inl i)))) fromAssumption
+
+
+
+
+      -- some more names to make the transport readable
+      pbPr₁PathP : PathP (λ i → C [ DLRan F .F-ob (⋁β++γ≡x∨y i) , DLRan F .F-ob (⋁γ≡y i) ])
+                         (pbPr₁ ⋁Pullback) (DLRan F .F-hom (hom-∨₂ L C x y))
+      pbPr₁PathP i = DLRan F .F-hom
+                       (isProp→PathP {B = λ i → (⋁γ≡y i) ≤ (⋁β++γ≡x∨y i)}
+                                     (λ _ → is-prop-valued _ _)
+                                     (subst (⋁ γ ≤_) (sym (⋁Split++ β γ)) (∨≤LCancel _ _))
+                                     (hom-∨₂ L C x y) i)
+
+      pbPr₂PathP : PathP (λ i → C [ DLRan F .F-ob (⋁β++γ≡x∨y i) , DLRan F .F-ob (⋁β≡x i) ])
+                         (pbPr₂ ⋁Pullback) (DLRan F .F-hom (hom-∨₁ L C x y))
+      pbPr₂PathP i = DLRan F .F-hom
+                       (isProp→PathP {B = λ i → (⋁β≡x i) ≤ (⋁β++γ≡x∨y i)}
+                                     (λ _ → is-prop-valued _ _)
+                                     (subst (⋁ β ≤_) (sym (⋁Split++ β γ)) (∨≤RCancel _ _))
+                                     (hom-∨₁ L C x y) i)
+
+      squarePathP : PathP (λ i → pbPr₁PathP i ⋆⟨ C ⟩ cospanPath i .s₁
+                               ≡ pbPr₂PathP i ⋆⟨ C ⟩ cospanPath i .s₂)
+                          (pbCommutes ⋁Pullback) (Fsq L C x y (DLRan F))
+      squarePathP = toPathP (isSetHom C _ _ _ _)
```

Comment:
Combine with the other lemma

Reply:
Added a few more lines, so we can be absolutely sure we get the desired result...